metal/cloud.go

```diff
@@ -31,8 +31,6 @@ type cloud struct {
 	config                      Config
 	instances                   cloudprovider.InstancesV2
 	loadBalancer                cloudprovider.LoadBalancer
-	metro                       string
-	facility                    string
```

Comment:
These were unused

Reply:
Yes. Same as the last. These should have gone with the last big PR.